### PR TITLE
fix: Properly retry fuzzy search test commands

### PR DESCRIPTION
**Pull request by agg23**
Fixes flaky `SpecList` test issues.

### CODE REVIEW

File: packages/runner-ct/cypress/component/SpecList.spec.tsx [hunk 1]
```diff
@@ -139,7 +145,7 @@ describe('SpecList', () => {
 
     mount(<Subject />, { styles })
 
-    cy.get('[placeholder="Find spec..."').click()
+    cy.get('[placeholder="Find spec..."]').click()
```
Comment (elevatebart) [inline packages/runner-ct/cypress/component/SpecList.spec.tsx:148]: Is this only change with significance?

**Review by elevatebart — APPROVED** [review]
LGTM

**Review by jennifer-shehane — APPROVED** [review]
I'm all for removing an arbitrary cy.wait(). Yeah, hopefully this should help.

File: packages/runner-ct/cypress/component/SpecList.spec.tsx (hunk 1 above)
Comment (agg23) [reply packages/runner-ct/cypress/component/SpecList.spec.tsx:148]: No, actually this doesn't fix any known issue. The change at the bottom (from `.contains` to `.should('contain', input)`) is the real fix, as for some reason the retry logic is different between those commands.

**Review by lmiller1990 — APPROVED** [review]

**Pull request merged**